kola/tests/misc/tls.go

```diff
@@ -0,0 +1,34 @@
+package misc
+
+import (
+	"fmt"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+)
+
+var (
+	urlsToFetch = []string{
+		"https://www.example.com/",
+		"https://www.wikipedia.org/",
+		"https://start.fedoraproject.org/",
+	}
+)
+
+func init() {
+	register.Register(&register.Test{
+		Run:              TestTLSFetchURLs,
+		ClusterSize:      1,
+		Name:             "coreos.tls.fetch-urls",
+		ExcludePlatforms: []string{"qemu"}, // Networking outside cluster required
+	})
+}
+
+func TestTLSFetchURLs(c cluster.TestCluster) {
+	m := c.Machines()[0]
+
+	for _, url := range urlsToFetch {
+		c.MustSSH(m, fmt.Sprintf("curl %s", url))
```

Comment:
We should add a retry amount, timeout and consider adding silent & show-error.

Both silent & show-error will still output in case of an error but will disable the progress bar.

`curl -s -S -m 30 --retry 2 %s`

Reply:
Thank you, will add these in.